ca.go

```diff
@@ -22,10 +22,9 @@ type CA struct {
 	Signer ssh.Signer
 }
 
-// Sign an SSH Certificate template (with `Key` set), and return the base64
-// encoded ssh key entry (something like `ssh-*-cert`) that the user can
-// import.
-func (s CA) Sign(template ssh.Certificate) ([]byte, error) {
+// Sign an SSH Certificate template (with `Key` set), and return the
+// certificate.
+func (s CA) Sign(template ssh.Certificate) (*ssh.Certificate, error) {
```

Comment:
The main reason why I did this was to match the API provided by `crypto/x509.CreateCertificate`. I'm fine changing this - it makes things a bit simpler.

Reply:
Yeah, my thought was that it made sense for `ca.go` as a more general purpose library, but at this point we've stripped it down a bit.

Reply:
Agree. Thanks!